### PR TITLE
Add Django 2.0 to the build matrix

### PR DESCRIPTION
**Pull request by browniebroke**
Django 2.0 is out!

- [x] Run tests against it
- [x] Fix potentail issues?

### CODE REVIEW

**Comment by evansd** [comment]
Thanks! Been on my todo list forever but have been swamped with other stuff :) Let's see how the tests do...

File: tox.ini [hunk 1]
```diff
@@ -29,6 +30,7 @@ deps =
   django19: Django>=1.9,<1.10
   django110: Django>=1.10,<1.11
   django111: Django>=1.11,<2.0
+  django111: Django>=2.0,<2.1
```
Comment (evansd) [inline tox.ini]: This should be django20

File: .travis.yml [hunk 2]
```diff
@@ -31,6 +36,9 @@ matrix:
       env: lint
       install: travis_retry pip install flake8==3.3.0
       script: flake8 --show-source
+  exclude:
+    - python: 2.7
```
Comment (evansd) [inline .travis.yml]: We need to exclude PyPy too

Comment (browniebroke) [reply .travis.yml]: I'm surprised, I didn't know they stopped supporting PyPy...

Comment (evansd) [reply .travis.yml]: PyPy in this context means the Python 2.7 version. I suppose what really ought to do is to add a PyPy3 environment as well.

Comment (browniebroke) [reply .travis.yml]: Ah yes of course! Yes, but _I think_ Django doesn't support for PyPy3 officially (that might no longer be true though...)

**Comment by evansd** [comment]
That was easier than expected! Thanks :)

**Comment by browniebroke** [comment]
Me too! Cheers.

**Pull request merged**